proto.proj

```diff
@@ -6,9 +6,6 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <Projects Include="src\fsharp\FSharp.Build\FSharp.Build.fsproj">
-      <AdditionalProperties Condition="'$(OS)' == 'Unix'">TargetFramework=netcoreapp3.1</AdditionalProperties>
-    </Projects>
     <Projects Include="src\fsharp\fsc\fsc.fsproj">
       <AdditionalProperties Condition="'$(OS)' == 'Unix'">TargetFramework=netcoreapp3.1</AdditionalProperties>
```

Comment:
can this now be moved to the upper property group unconditionally?

Reply:
hmm, this is proto.proj, was this intended?

Reply:
it was